ansible/playbook_Test_Scripts/Test_Script

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+printf "Enter the URL:  "			#User Input section
```

Comment:
Instead of asking the user it's often better to take in input arguments. That way other scripts can call this script with no human interaction.